sockeye/encoder.py

```diff
@@ -281,13 +281,15 @@ def __init__(self,
                  vocab_size: int,
                  num_embed: int,
                  dropout: float,
+                 frozen: bool = False,
```

Comment:
I find this name a bit confusing (given that we have the hidden member in configs called _frozen). 
But more general, I wonder if we should implement fixing certain parameters during training with the functionality in the Module API (http://mxnet.incubator.apache.org/api/python/module/module.html) and its member `fixed_param_names`, which right now defaults to None.
This would allow a more general way of skipping optimization for certain parameter sets.

Reply:
My original name was "freeze" which lead to a funny error message about bool not being callable... I'm open to naming suggestions.

fixed_param_names is also a good way to go (I used it in another branch), but we'll probably should discuss the user interface for it.

Reply:
How about the simplest interface possible, an additional CLI argument (next to `--allow-missing-params` that takes a list of strings that correspond to the names of the parameters to be fixed during training. We then simply pass this to the Module constructor. This is transparent, does not touch the symbol generating code (encoder.py) but is limited to the training code, and users only need to know the parameter names, which they can get easily by starting the training and looking at the log.